blocks/card-grid/card-grid.css

```diff
@@ -0,0 +1,97 @@
+.section .card-grid-wrapper {
```

Comment:
We already have the layout implemented in card, see https://main--24life--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/block-library/blocks/card&index=2
It does not yet do large on the right side yet. 

We could reuse that feature here. Then this block would populate the cards from the links, and control the layout, but could be simpler? WDYT?

Reply:
Yes. I didn't realize it was there.